build_helper.sh

```diff
@@ -131,15 +146,32 @@ function setup_folly() {
       exit 1
     fi
   fi
+
+  if [ "$Platform" = "Mac" ]; then
+    # Homebrew installs OpenSSL in a non-default location on  MacOS >= Mojave
+    # 10.14 (because MacOS has its own SSL implementation).  If we find
+    # Homebrew's OpenSSL pkgconfig file, set it so that cmake will find/use it.
+    dir=/usr/local/opt/openssl/lib/pkgconfig
```

Comment:
Here I think it's simpler to just add sth as:
OPENSSL_ROOT_DIR=/usr/local/opt/openssl
OPENSSL_LIB_DIR=/usr/local/opt/openssl/lib/

and then pass these variables to cmake build cmd if platform is Mac (e.g. https://github.com/facebookincubator/fizz/blob/master/fizz/mac-build.sh#L26) 

We can also accept these as optional parameters for this script.

Reply:
I also don't have a strong opinion on this one, but I thought using pkg-config was preferred (i.e., query the `.pc` file) rather than try to assume the lib/include directory.

Reply:
Ok, I updated all the others.  Let me know if you want me to update this one, too (I left it as prepending `PKG_CONFIG_FILE` just because that's my natural bias, but if you want it the other way, that's fine).

Reply:
Sure. I'll test it internally and will merge it back. Thank you for your effort!

Reply:
@jsquyres I tried this on a clean repo on Mac. And it still fails to find the OPENSSL. So opting the approach similar to https://github.com/facebookincubator/fizz/blob/master/fizz/mac-build.sh#L26 is probably going to be a simpler fix.

CMake Error at /usr/local/Cellar/cmake/3.12.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_INCLUDE_DIR)

Reply:
Weird; I wonder why that doesn't work for you.  🤷‍♂ 

It looks like only `OPENSSL_ROOT_DIR` is necessary (we don't need `OPENSSL_LIB_DIR`, as shown in the Fizz `mac-build.sh` script).  I pushed an update; give it a whirl.